### DISCUSSION TITLE
search input box render bug

### DISCUSSION THREAD

**Discussion by Jack-An** [community/community#29785] [Projects and Issues]
![image](https://user-images.githubusercontent.com/31822423/185088633-b2d7b2b5-2c36-40ab-9ba6-e4792d0c9c56.png)

**Comment by jfullstackdev** [comment]
I saw the same report and one Staff already responded

**Reply by queenofcorgis** [reply]
@jdevstatic do you mind sharing the link so @Jack-An can follow along if possible?

**Reply by jfullstackdev** [reply]
fortunately, I found it haha 

this one 

https://github.com/orgs/community/discussions/29769

**Reply by queenofcorgis** [reply]
Thank you!
